.devcontainer/devcontainer.json

```
@@ -0,0 +1,77 @@
+{
+    "name": "prototyping",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": {
+            "POETRY_VERSION": "1.8.*"
+        }
+    },
+    "forwardPorts": [3000],
+    "initializeCommand": "bash .devcontainer/initializeCommand.sh",
+    "onCreateCommand": "bash .devcontainer/onCreateCommand.sh",
+    "updateContentCommand": "bash .devcontainer/updateContentCommand.sh",
+    "postStartCommand": "git config --global --add safe.directory ${containerWorkspaceFolder}",
+    "containerEnv": {
+        "WORKSPACE": "${containerWorkspaceFolder}"
```

```suggestion
        "WORKSPACE": "${containerWorkspaceFolder}"
        "XDG_CONFIG_HOME": "/home/gitpod/.config"
```

This will fix the error when the container build